Makefile

```diff
@@ -11,19 +11,19 @@ capdef_file = ./irc/caps/defs.go
 
 all: build
 
-install:
+install: build
 	go install -v -ldflags "-X main.commit=$(GIT_COMMIT) -X main.version=$(GIT_TAG)"
 
-build:
+build: ${capdef_file}
```

Comment:
I'd rather we didn't, this just slows down the build in the normal case?

Reply:
it's only a stat() call...

Reply:
(Make won't rebuild a target if a file with the same name already exists and is more recent than its dependencies)

Reply:
Oh right, I normally don't use this feature of make (I normally do 100% phony targets). I would prefer if we didn't introduce it because I don't understand it very well :-|